addons/l10n_din5008/views/res_config_settings_views.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="res_config_settings_view_form" model="ir.ui.view">
+        <field name="name">res.config.settings.view.form.inherit.base.vat</field>
+        <field name="model">res.config.settings</field>
+        <field name="inherit_id" ref="account.res_config_settings_view_form"/>
+        <field name="arch" type="xml">
+            <setting id="display_invoice_tax_company_currency" position="after">
+                <setting id="position_column"
+                         help="Enable to show a 'Position' column in invoice and sales reports."
+                         title="If this checkbox is ticked, a 'Position' column will be displayed on invoices and sales reports.">
+                    <field name="has_position_column"/>
+                </setting>
+            </setting>
+        </field>
+    </record>
```

Comment:
it will be put for every company even the one that don't need din5008 once the module is installed, we need to check if this is a problem